tools/sql/handler.go

```diff
@@ -170,10 +170,13 @@ func doDropDatabase(p ConnectParams, name string) {
 	p.Database = ""
 	conn, err := NewConnection(&p)
 	if err != nil {
-		handleErr(err)
+		logErr(err)
```

Comment:
I think the logic here should be 
```
if err = handler(err); err != nil {
    logErr(err)
}
```
and logging should be removed from handleErr().
Same below.

Reply:
That a good observation but `handleErr` here is:
```
func handleErr(err error) error {
	log.Println(err)
	return err
}
```

That's why I write that way in PR. Will keep it like that